src/utils/AppRestart.ts

```diff
@@ -1,19 +1,13 @@
 import { Platform } from 'react-native'
-import RNExitApp from 'react-native-exit-app'
-import { RestartAndroid } from 'react-native-restart-android'
-import { AppEvents } from 'src/analytics/Events'
+import RNRestart from 'react-native-restart'
 import AppAnalytics from 'src/analytics/AppAnalytics'
+import { AppEvents } from 'src/analytics/Events'
 import Logger from 'src/utils/Logger'
 
 export const RESTART_APP_I18N_KEY = Platform.OS === 'android' ? 'restartApp' : 'quitApp'
 
 export function restartApp() {
   AppAnalytics.track(AppEvents.user_restart)
   Logger.info('utils/AppRestart/restartApp', 'Restarting app')
-  if (Platform.OS === 'android') {
-    RestartAndroid.restart()
-  } else {
-    // We can't restart on iOS, so just exit ;)
-    RNExitApp.exitApp()
-  }
+  RNRestart.restart()
```

Comment:
does this still only exit on iOS or now it can restart? Wonder if we can update the i18n key? Or at least leave a comment if this still only can exit the app on iOS

Reply:
It restarts on iOS now so I'll update the key.

Reply:
Fixed in 485b963bbed0adc2e378db8a2b8c55ef5b90162d!